monai/transforms/spatial/array.py

```diff
@@ -667,7 +667,9 @@ def __init__(
     def randomize(self):
         self._do_transform = self.R.random_sample() < self.prob
         if hasattr(self.min_zoom, "__iter__"):
-            self._zoom = (self.R.uniform(l, h) for l, h in zip(self.min_zoom, self.max_zoom))
+            # TODO: Review the types here
```

Comment:
This suppression is to allow for clean static checking for type hints.  This case will need to be revisited in the future, but suppressing this difficult type hint case temporarily in order to efficiently address more beneficial issues first.